Elmah.Io.ElasticSearch/ElasticClientSingleton.cs

```diff
@@ -18,9 +18,13 @@ public class ElasticClientSingleton : IDisposable
         private static ElasticClientSingleton _instance;
         public IElasticClient Client;
 
-        private ElasticClientSingleton()
+        private ElasticClientSingleton(IDictionary config = null)
         {
-            var config = ReadConfig();
+            if(ReferenceEquals(config, null))

```

Comment:
@TrymBeast in C# that's really not necessary and it would be a lot of code everywhere else.